main.rb

```diff
@@ -0,0 +1,69 @@
+require './app'
+
+class Main
+  def initialize
+    @app = App.new
+  end
+
+  puts 'Welcome to School Library App!'
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def run_option
+    loop do
+      display_menu_options
+      option = gets.chomp
+
+      case option
+      when '1' then list_all_books
+      when '2' then list_all_people
+      when '3' then create_person
+      when '4' then create_book
+      when '5' then create_rental
+      when '6' then list_rentals
+      when '7'
+        puts 'Thank you for using our library ¯\^-^/¯'
+        return
+      else
```

Comment:
currently its impossible to start the app because of the below error

![image](https://github.com/Patfarmurs/oop-school-library/assets/60612329/ad6d924c-48d2-4f64-8822-9f11487fdd99)

to fix this error please remove the piece of code highlighted below in the person.rb file 👍🏾 
![image](https://github.com/Patfarmurs/oop-school-library/assets/60612329/afc6720e-b6c6-4ac6-8dd3-5479bdc45793)